amundsen_application/api/search/v0.py

```diff
@@ -57,6 +71,73 @@ def _validate_search_term(*, search_term: str, page_index: int) -> Optional[Resp
     return None
 
 
+@search_blueprint.route('/table_qs', methods=['POST'])
+def search_table_qs() -> Response:
+    # TODO (ttannis): This method is in development and will be cleaned up in a future PR
+    request_json = request.get_json()
+
+    search_term = get_query_param(request_json, 'term', '"term" parameter expected in request data')
+    page_index = get_query_param(request_json, 'pageIndex', '"pageIndex" parameter expected in request data')
+    filters = request_json.get('filters', {})
+
+    filter_payload = {}
+    valid_categories = ['column', 'database', 'schema', 'table', 'tag']
+    for category in valid_categories:
+        values = filters.get(category)
+        if type(values) == str and len(values) > 0:
+            filter_payload[category] = [values, ]
+        if type(values) == dict:
```

Comment:
What's the use case for filter values being a dictionary? I was under the assumption filter values would either be a string or an array of strings.

Reply:
Matching the current `FilterReducerState`, as the code just sends over the pieces of state and delegates the conversion to this method. I see you have some further comments about the state so that could always change what happens here. This method is essentially in development because of both potential amundsensearch api changes *and* redux state changes.

Reply:
Ah also worth noting the dictionary shape was ideal for knowing if a multi-select option was checked. 

Here is an example `FilterReducerState`:
```
{
   'database': { 'bigquery': true, 'hive': true }
   'tag': 'ttannis'
}
```

Here is what amundsensearch needs:
```
{
   'database':  [ 'bigquery', 'hive']
   'tag': ['ttannis']
}
```

While it requires a conversion somewhere or the other (whether in Flask or Redux), the concept of modeling the application state based on what is best for the application's use cases for accessing that state is what guided this choice.